frontend/src/components/CreateInstanceForm.vue

```diff
@@ -85,6 +94,8 @@
             :placeholder="
               state.instance.engine == 'SNOWFLAKE'
                 ? $t('instance.your-snowflake-account-name')
+                : state.instance.engine == 'SPANNER'
+                ? 'projects/<projectID>/instances/<instanceID>'
                 : $t('instance.sentence.host.snowflake')
```

Comment:
Probably not introduced by this PR. But this i18n resource key is quite confusing.

Reply:
+1 Let's fix it in a follow-up PR then.